src/api/v1/files.py

```diff
@@ -96,21 +100,22 @@ def get_file_content(
 
     html_content = f"""
     <html style="background:{background_color}">
-        <pre style="color:{font_color};padding:10px;white-space: pre-wrap;">{ html_source_content }</pre>
+        <pre style="color:{font_color};padding:10px;white-space: pre-wrap;">{html_source_content}</pre>
     </html>
     """
     # return content
     return HTMLResponse(content=html_content, status_code=200)
 
 
 # Download file
-@router.post("/{file_id}/download")
+@router.get("/{file_id}/download")
```

Comment:
I just realize we don't use this method anymore. It was unefficient for the browser (read the whole file in memory before download happen etc). We use the download_stream method instead. Let's consider removing this endpoint in a followup PR.